### PR TITLE
#163392422 Admin can delete meetup 

### PR DESCRIPTION
**Pull request by Bakley**
### What does this PR do?
- Admin can delete a meetup
### Description of Task to be completed?
- Have the following endpoint working
```
DELETE /api/v2/meetups/1
```
### How should this be manually tested?
- After cloning the repo
- Create a database in postgreSQL
- Run
```
export FLASK_APP=run.py
```
```
export FLASK_ENV=development
```
```
export DEV_DB_URI="dbname='<databasename>' host='localhost' port='5432' user='postgres' password='password'"
```
```
flask run
```
- Use postman to test the endpoint
### What are the relevant pivotal tracker stories?
#163392422 

### Screenshots
delete_meetup

### CODE REVIEW

File: app/api/v2/models/meetup_models.py [hunk 1]
```diff
@@ -47,3 +47,14 @@ def get_all_meetups(self):
             "upcoming_meetup": response
         }
         return payload
+
+    def delete_a_specific_meetup(self, id):
+        """Admin can delete a meetup"""
+        fetch_query_for_delete = """SELECT * FROM meetups
+        WHERE meetupId={}""".format(id)
+        response = QuestionerDb.retrieve_one(fetch_query_for_delete)
+        print("Delete this", response)
```
Comment (lennykioko) [inline app/api/v2/models/meetup_models.py:56]: What's the purpose of this line?

Comment (Bakley) [reply app/api/v2/models/meetup_models.py:56]: To query if the meetup first exists. So that I can go ahead and delete it

Comment (Bakley) [reply app/api/v2/models/meetup_models.py:56]: If the meetup doesn't exist, returns a false which inform the admin that no meetup of the id being queried is available

**Comment by blairt001** [comment]
Looks great

**Pull request merged**